docker/autorun-amc2moodle.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+## script to run amc2moodle or moodle2amc automatically in a specific folder
+
+# monitored folder
+MONITOR_DIR=/tmp/work
```

Comment:
Perhaps we may use a docker env variable in the Dockerfile, to avoid this hard coded path
```
ENV MONITOR_DIR=/tmp/work
```
and have all config in the same place.